### PR TITLE
Make all API endpoints and response formatter driven from tenant JSON…

### PR DESCRIPTION
**Pull request by BhagyashreeBhoyar**
… configuration.

### CODE REVIEW

File: src/utils/apis/formatters/YJSG-Bhopal/headerFormatters.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const headerParamsFormatter = (headers, additionalData) => {
```
[the remark this answers is not in the document]
Comment (BhagyashreeBhoyar) [reply src/utils/apis/formatters/YJSG-Bhopal/headerFormatters.js]: Done.

File: src/utils/apis/formatters/YJSG-Bhopal/index.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+export const yjsgBhopal = {
+  header: {
+    updateMember: headerParamsFormatter,
+  },
+  request: {
+    createMember: formatCreateMemberDataPayload,
+    updateMember: formatUpdateMemberDataPayload,
+  },
+  response: {
+    createMember: response => response,
```
[the remark this answers is not in the document]
Comment (BhagyashreeBhoyar) [reply src/utils/apis/formatters/YJSG-Bhopal/index.js]: Done.

File: src/utils/apis/formatters/YJSG-Indore/headerFormatters.js [hunk 3]
```diff
@@ -1,7 +1,22 @@
...
+  const { secretKey } = additionalData;
+  return ({
+    secretKey,
+  });
+};
+
+export const markMemberAPIHeaderFormatter = (header, additionalData) => {
+  const { secretKey } = additionalData;
+  return ({
+    'Content-type': 'application/json',
```
Comment (pepcus-pratik) [inline src/utils/apis/formatters/YJSG-Indore/headerFormatters.js]: Mention this in the api.json file

Comment (pepcus-pratik) [reply src/utils/apis/formatters/YJSG-Indore/headerFormatters.js]: @BhagyashreeBhoyar address this in the next PR

Comment (BhagyashreeBhoyar) [reply src/utils/apis/formatters/YJSG-Indore/headerFormatters.js]: Done.

**Comment by pepcus-pratik** [comment]
For https://github.com/Pepcus/yjsgUI/issues/184

**Pull request merged**